lib/deppack.js

```diff
@@ -0,0 +1,289 @@
+'use strict';
+const sysPath = require('path');
+const fs = require('fs');
+const detective = require('detective');
+const browserResolve = require('browser-resolve');
+const requireDefinition = require('commonjs-require-definition');
+const each = require('async-each');
+
+const shims = {
+  'assert': {},
```

Comment:
We can just make a list of items and assign an empty object to each of those. 30 less LOCs.